test/scripts/agent-images/create_application_image.sh

```
@@ -1,31 +1,52 @@
-   podman build "${CACHE_FLAGS[@]}" \
-   	--build-arg SOURCE_GIT_TAG=${SOURCE_GIT_TAG:-$("${SCRIPT_DIR}/../../../hack/current-version")} \
-   	--build-arg SOURCE_GIT_TREE_STATE=${SOURCE_GIT_TREE_STATE:-$( ( ( [ ! -d ".git/" ] || git diff --quiet ) && echo 'clean' ) || echo 'dirty' )} \
-   	--build-arg SOURCE_GIT_COMMIT=${SOURCE_GIT_COMMIT:-$(git rev-parse --short "HEAD^{commit}" 2>/dev/null || echo "unknown")} \
-   	-f "${SCRIPT_DIR}"/Containerfile-sleep-app-"${img}" -t localhost:5000/sleep-app:${img} .
-   podman tag "localhost:5000/sleep-app:${img}" "${FINAL_REF}"
-   podman push "${FINAL_REF}"
-done
+current_version() {
+  local version=""
+  if [[ -x "${CURRENT_VERSION_SCRIPT}" ]]; then
+    version=$((cd "${ROOT_DIR}" && "${CURRENT_VERSION_SCRIPT}") 2>/dev/null || true)
```

Shellcheck (0.11.0)

[error] Shells disambiguate $(( differently or not at all. For $(command substitution), add space after $( . For $((arithmetics)), fix parsing errors.

(SC1102)